<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPurchaseProductOrderView</class>
 <widget class="QWidget" name="AddPurchaseProductOrderView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new purchase product order</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Product</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="productLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QToolButton" name="chooseProductToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Product Origin</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="productOriginComboBox">
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>CST ICMS</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="icmsComboBox">
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QComboBox" name="ipiComboBox">
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>CST PIS</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>CST COFINS</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="pisComboBox">
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="3" colspan="2">
       <widget class="QComboBox" name="cofinsComboBox">
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Price Value (R$)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="priceDoubleSpinBox">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QLineEdit" name="quantityLineEdit">
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>CST IPI</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>CFOP</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="cfopLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QToolButton" name="cfopPushButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="icmsDoubleSpinBox">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="pisDoubleSpinBox">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
      </item>
      <item row="5" column="5">
      <widget class="QDoubleSpinBox" name="ipiDoubleSpinBox">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
      </item>
      <item row="7" column="5">
      <widget class="QDoubleSpinBox" name="cofinsDoubleSpinBox">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>74</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okPushButton">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
